apache/vhosts.conf

```diff
@@ -8,7 +8,7 @@
     </Directory>
 
     WSGIDaemonProcess rdmo user=rdmo group=rdmo processes=2 \
-        python-path=/vol/rdmo-app:/vol/ve/lib/python3.5/site-packages
+        python-path=/vol/rdmo-app:/vol/ve/lib/python3.7/site-packages
```

Comment:
I think there's no need to include the expected python version in the `python-path`. At the point in the future where `apk` will install a different python version, the problem will arise again. Have a look at my commit https://github.com/JKatzwinkel/rdmo-docker-compose/commit/b1180465fc9cc71048111c64e7d2f8699693eb04 where I omitted the `site-packages` path entirely, which solved the issue for me. Also, I specified the `python-home` path as they suggest in the wsgi module's documentation.

Reply:
Yes, thanks, `python-home` is the way to go. I will try to fix this tomorrow.

Reply:
ok sry I didn't see that you already had all of that figured out in the issue @suchmaske referenced